posts/dask-summit-takeaway.md

```diff
@@ -24,6 +24,23 @@ tags: dask, conference, open-science
 - STAC (spatio-temporal asset catalog) can be helpful when working with satellite imagery, making it easy to search for specific datasets
   - Similar to intake, able to search spatially as well
 
+### [Pangeo Session Talk Links](https://summit.dask.org/schedule/presentation/1/pangeo/)
+
+- Chelle: notebook can be found in folder AWS-notebooks
+  - [Jupyter Notebooks](https://github.com/pangeo-gallery/osm2020tutorial)
+- Julius:
+  - [Slides](https://speakerdeck.com/jbusecke/dask-and-the-ocean-death-zones-lessons-from-a-real-life-earth-science-workflow-with-a-fullish-pangeo-stack)
+  - [CMIP6 Prepocessing](https://github.com/jbusecke/cmip6_preprocessing)
+  - [Oxygen Minimum Zones Paper Preprint](https://www.essoar.org/doi/10.1002/essoar.10507050.1)
+- Rob
+  - [Info about STAC specification](https://stacspec.org)
+- Deepak:
+  - [Dask Groupby Repository](https://github.com/dcherian/dask_groupby)
+- Haiying:
+  - [Pangeo Benchmarking Repository](https://github.com/pangeo-data/benchmarking)
+- Damien:
+  - [Software Carpentries Lessons](https://github.com/carpentrieslab/python-aos-lesson)
+
 ## [Growth and History of Anaconda + Numpy + SciPy + Dask](https://zoom.us/rec/share/_KQiagnZBnW0xPIHAHZZs553JuFsJXfkQpdwUmTca7-QE91safAQCefSYr6-Kz3R.EDHHYf_864Dw3v8Q?startTime=1621515755000)
```

Comment:
I wouldn't share these Zoom links since they are likely going to disappear once the recordings are posted on YouTube. I recommend waiting until early next week for the YouTube Links.

Reply:
Any idea of the timeline for the release of these videos?

Reply:
2 months based on an email I received: 
> Video recordings of all the sessions can be watched at summit.dask.org/events (no password required). In the next two months all the videos will be uploaded to the Dask YouTube channel. 

That events page lists the zoom links so those should work for a while. I don't think this is a blocker now.

Reply:
Sounds good - yeah I figured the zoom links would be good for now (they don't require a password which is nice!)